app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/util/Triplet.java

```diff
@@ -0,0 +1,34 @@
+package com.epfl.neighborfood.neighborfoodandroid.util;
+
+// Triplet class
+public class Triplet<U, V, T>

```

Comment:
It's good that you added this in a separate package, this could come in handy for other use cases